scripts/shared/clusters.sh

```diff
@@ -1,9 +1,17 @@
 #!/usr/bin/env bash
 
+## Kubernetes version mapping, as supported by kind ##
+# See the release notes of the kind version in use
+declare -A kind_k8s_versions
+kind_k8s_versions[1.17]=1.17.17
+kind_k8s_versions[1.18]=1.18.15
+kind_k8s_versions[1.19]=1.19.7
+kind_k8s_versions[1.20]=1.20.2
+
 ## Process command line flags ##
 
 source ${SCRIPTS_DIR}/lib/shflags
-DEFINE_string 'k8s_version' '1.20.2' 'Version of K8s to use'
+DEFINE_string 'k8s_version' '1.20' 'Version of K8s to use'
```

Comment:
Side comment to this patch, the change looks good:

Oh, I see this is 1.20 by default. I wonder if in the consuming projects we should leave the 1.20 version empty. At least that gives us the indication of which version was it tested with so may be it's better to keep it